packages/web/src/ui/views/options.js

```diff
@@ -6,60 +6,60 @@ module.exports = function options (state, i18n) {
     return html`<option value='${language.code}' selected=${selected}>${i18n.translate(language.fullName)}</option>`
   })
   const shortcuts = require('./shortcuts')(state, i18n)
+
   return html`
-<section id='options' style='visibility:${state.activeTool === 'options' ? 'visible' : 'hidden'}; color:${state.themes.themeSettings.secondaryTextColor}'>   
-  <br>
+  <section id='options' style='visibility:${state.activeTool === 'options' ? 'visible' : 'hidden'}; color:${state.themes.themeSettings.secondaryTextColor}'>   
+
   <fieldset>
-    <legend> <h3> ${i18n`language`}</legend>
+    <legend> <h3> ${i18n`options.language.h3`} </h3> </legend>
     <select id='languageSwitcher'>
       ${languages}  
     </select>
   </fieldset>
 
   <fieldset>
-    <legend> <h3> ${i18n`theme`} </h3> </legend>
+    <legend> <h3> ${i18n`options.theme.h3`} </h3> </legend>
```

Comment:
not repeating , but same question as above about the use of xxxDOTxxxDOT for translation strings

Reply:
I’m using the ‘.’ to separate words, as this reads easier.